geometry/circle.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+
+def calculate_area(r):
+    """
+    Calculates the area of a circle.
+
+    Parameters
+    ----------
+    r : float or array
+        The radius of a single circle or multiple circles.
+
+    Returns
+    -------
+    area : float or array
+        The calculated area/s.
+        
+    Examples
+    --------
+    >>> calculate_area(1)
+    3.141592653589793
+    
+    """
+    if np.any(r < 0): 
+        raise ValueError("Radius inputs should be non-negative")
+    
+    area = np.pi * r **2
+    return area
+
+
+def calculate_circ(r):
+    """
+    Calculates the circumference of a circle.
+
+    Parameters
+    ----------
+    r : float or array
+        The radius of a single circle or multiple circles
```

Comment:
This sentence needs a period at it's end.

```suggestion
        The radius of a single circle or multiple circles.
```